docs/source/introduction.rst

```diff
@@ -130,26 +132,40 @@ range plotting possibilities with example code.
 
 ----
 
-**Old versions**
-----------------
-
-Since version 3.0.0 (released 2019-10-01), cf is for Python 3 only and
-there are :ref:`incompatible differences between versions 2.x and 3.x
-<two-to-three-changes>` of cf.
+**Performance**
+---------------
 
-Scripts written for version 2.x but running under version 3.x should
-either work as expected, or provide informative error messages on the
-new API usage. However, it is advised that the outputs of older
-scripts are checked when running with Python 3 versions of the cf
-library.
-
-For version 2.x documentation, see the :ref:`releases <Releases>`
-page.
+As of version 3.14.0 (released 2023-01-31), cf uses :ref:`Dask
+<Performance>` for all of its data manipulations, which provides lazy,
+parallelised, and out-of-core computations of array operations.
 
 ----
-   
-.. [#cfdm] Hassell, D., Gregory, J., Blower, J., Lawrence, B. N., and
-           Taylor, K. E.: A data model of the Climate and Forecast
-           metadata conventions (CF-1.6) with a software
-           implementation (cf-python v2.1), Geosci. Model Dev., 10,
-           4619-4646, https://doi.org/10.5194/gmd-10-4619-2017, 2017.
+
+**References**
+--------------
+
+Eaton, B., Gregory, J., Drach, B., Taylor, K., Hankin, S., Caron, J.,
```

Comment:
Can we maybe bullet point these instead? I ask mainly because the spacing and new lines seem to be preserved in the built docs and it looks slightly odd (only if you stare at it for a bit, which clearly nobody will do, but hey...):

![breaks-docs](https://user-images.githubusercontent.com/30274190/215633287-8b0d48c6-5153-475a-94a6-c760da766269.png)

Reply:
Oh - I thought it was nice to mimic the reference style of published journals, e.g. GMD and JOSS, which uses haning paragraphs and new lines just like this ...